toplevel/toplevel.go

```diff
@@ -12,6 +12,9 @@ import (
 var (
 	configs  = make(map[string]Configuration)
 	configsM sync.RWMutex
+	// add a new map variable here. the keys will be policy names and the values will be `delete` or `update`
+	// your discretion for how to store delete/update value
+	// could be an enum?
```

Comment:
Will I need to use a mutex here? Seeing that maps aren't concurrency safe

Reply:
I don't believe so. the existing mutex is because `RegisterConfiguration` is called within the init of every toplevel. The new var will only be processed by a single toplevel, at the time it is reconciling

Reply:
Gotcha, thanks